be/src/formats/json/struct_column.cpp

```diff
@@ -37,8 +37,19 @@ Status add_struct_column(Column* column, const TypeDescriptor& type_desc, const
             const auto& field_type_desc = type_desc.children[i];
 
             auto field_column = struct_column->field_column(field_name);
-            simdjson::ondemand::value field_value = obj.find_field_unordered(field_name);
-            RETURN_IF_ERROR(add_nullable_column(field_column.get(), field_type_desc, name, &field_value, true));
+            simdjson::ondemand::value field_value;
+            auto err = obj.find_field_unordered(field_name).get(field_value);
```

Comment:
Will `obj.find_field_unordered(field_name)` be null?

Reply:
No